[doc] Some fixes for jit overview doc

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#28112 [doc] Some fixes for jit overview doc**

Summary:
att

Test Plan:
reading

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18173102](https://our.internmc.facebook.com/intern/diff/D18173102)

jerryzh168 committed:
[doc] Some fixes for jit overview doc

Summary:
att

Test Plan:
reading

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM commented on torch/csrc/jit/docs/OVERVIEW.md:
```diff
@@ -531,7 +531,7 @@ SugaredValues are how the compiler represents non-first class values during Grap
 SugaredValues are also how we interact with Python runtime during the compilation process. For instance, `math.pi` is resolved to 3.1415... by first resolving `math` to a SugaredValue representing accesses to Python modules (PythonModuleValue) whose `attr` function turns python numbers into  `prim::Constant` Nodes in the graph.
 
-Finally, normal Values are also represented by the SimpleValue SugaredValue in places where it is valid either a SugaredValue or a normal Value to appear.
+Finally, normal Values are also represented by the SimpleValue SugaredValue in places where it is valid either a SugaredValue or a normal Value will appear.
```
Should there be "that" before "either"?

ZolotukhinM reviewed (approved):
Thanks!

jerryzh168 committed:
Update on "[doc] Some fixes for jit overview doc"


Summary:
att

Test Plan:
reading

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18173102](https://our.internmc.facebook.com/intern/diff/D18173102)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[doc] Some fixes for jit overview doc"


Summary:
att

Test Plan:
reading

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18173102](https://our.internmc.facebook.com/intern/diff/D18173102)

[ghstack-poisoned]
